packages/now-next/src/index.ts

```diff
@@ -181,7 +192,19 @@ export const build = async ({
     if (!urls[entrypoint]) {
       console.log(`${name} Installing dependencies...`);
       await runNpmInstall(entryPath, ['--prefer-offline']);
-      const { forked, getUrl } = startDevServer(entryPath);
+
+      // The runtime env vars consist of the base `process.env` vars, but with the
+      // build env vars removed, and the runtime env vars mixed in afterwards
+      const runtimeEnv: EnvConfig = Object.assign({}, process.env);
+      const runtimeEnvKeys = Object.keys(meta.env || {});
```

Comment:
```suggestion
      const runtimeEnvKeys = new Set(Object.keys(meta.env || {}));
```

Might as well use a set since this is only used to check existence